src/Symfony/Component/Debug/Debug.php

```diff
@@ -53,6 +53,7 @@ public static function enable($errorReportingLevel = null, $displayErrors = true
             ini_set('display_errors', 1);
         }
         $handler = ErrorHandler::register();
+        $handler->setDefaultLogger(new BootstrappingLogger());
```

Comment:
and shouldn't we have a separate instance for each type of logging, to be sure we flush things to the right loggr when adding them later ?